tests/asts.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2020 PingCAP, Inc.
```

Comment:
move this file to `tests/e2e/tidbcluster` pkg and unexport identifies in this file.

Reply:
removed